packages/protocol/src/log-protocol.ts

```diff
@@ -3,4 +3,5 @@ import type { RuntimeLogEntry } from './data/log';
 export interface LogRequestTypes {
   data: [RuntimeLogEntry[], RuntimeLogEntry[]];
   clear: [];
+  newEntry: RuntimeLogEntry;
```

Comment:
this is not a requestType


```suggestion
export interface LogOnNotificationTypes {
  newEntry: RuntimeLogEntry;
}
  
```